docs/user-guide/ze-install-configuring-ze.md

```diff
@@ -0,0 +1,83 @@
+# Configuring Zowe Explorer
+
+:::info Required role: Developer
+:::
+
+Configure Zowe Explorer in the settings file so the extension performs according to your preferences.
+
+To access Zowe Explorer settings:
+
+1. Click the **Settings** icon at the bottom of the **Activity Bar**.
+2. Select the **Settings** option.
+3. Open the **Extension** option listed in the **Commonly Used** menu.
+4. Select **Zowe Explorer** to access its settings.
+5. Scroll the list to find the setting that needs modification.
+
+## Modifying creation settings for data sets, USS files, and jobs
+
+1. In Zowe Explorer settings, scroll to a data set, USS file, or job setting type.
+2. Click the setting's corresponding **Edit in settings.json** link.
+
+    This opens the `settings.json` file in an **Editor** tab. (The suggestions widget also opens if the functionality is enabled.)
+
+3. Edit the settings in the file as needed.
+4. Save the file to keep changes.
+
+    ![Configure Zowe settings](../images/ze/ZE-Configuration2.gif)
+
+## Modifying temporary file location settings
+
+Change the default folder location where temporary files are stored:
+
+   1. Navigate to Zowe Explorer settings.
+   2. Under the data set, USS, or jobs settings that you want to edit, click the **Edit in settings.json** link.
+   3. Modify the following definition in the file:
+
+      ```json
+      "zowe.files.temporaryDownloadsFolder": {
+          "folderPath": "/path/to/directory"
+        }
+      ```
+
+      Replace `/path/to/directory` with the new folder location.
+
+    4. Save the file to keep the change.
+
+## Modifying the `Secure Credentials Enabled` setting
+
+When environment conditions do not support the Zowe CLI built-in credential manager, change the `Secure Credentials Enabled` setting:
+
+ 1. Navigate to Zowe Explorer settings.
+ 2. Scroll to **Security: Secure Credentials Enabled**.
+ 3. Deselect the checkbox to disable secure credentials.
+
+    When disabled, if the `autoStore` setting in the `zowe.config.json` file is set to `true`, z/OS credentials are stored as text in the file.
+
+    If the `autoStore` setting is set to `false`, you are prompted for the missing credentials in VS Code. The credentials are stored and used for the duration of the session.
+
+## Setting confirmation requirements for submitting jobs
+
+Submitting the wrong job can risk potential problems on your server. This can happen when the user enters the wrong job or inadvertently selects the **Submit Jobs** option.
+
+To help prevent this, enable the option to require confirmation before submitting a job. Once enabled, a dialog window asking for user confirmation displays when **Submit Jobs** is selected.
+
+![Confirm Submit Job](../images/ze/ZE-SubmitJobConfirm.gif)
+
+To configure confirmation settings for submitting a job:
+
+1. On the VS Code menu bar, click **File**, **Preferences**, and click **Settings** to display the Settings editor.
+
```

Comment:
I noticed that we have two different methods outlined for navigating to the settings editor in VS Code. I don't have a preference for which method we suggest, but we might want to stick with one method for clarity/consistency